corgi/collectors/migrations/0006_auto_20220922_1834.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 3.2.15 on 2022-09-22 18:34
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("collectors", "0005_collectorrpmrepository"),
+    ]
+
+    operations = [
+        migrations.AddConstraint(
+            model_name="collectorrpm",
+            constraint=models.UniqueConstraint(
+                fields=("nvra", "srpm"), name="unique_collectorrpms"
+            ),
+        ),
+        migrations.AddConstraint(
+            model_name="collectorrpmrepository",
+            constraint=models.UniqueConstraint(
+                fields=("name", "content_set"), name="unique_collectorrpmrepos"
+            ),
+        ),
+    ]
```

Comment:
Maybe all three of these migrations can be squashed to just one? Or is there a reason to keep them separate?

Reply:
OK Done.